schema/Organization.re

```diff
@@ -0,0 +1,145 @@
+open Graphql_lwt;
+open GraphqlHelpers;
+
+module Project = {
+  type t = {
+    id: int,
+    name: string,
+    webhook: string,
+  };
+
+  let resolver =
+    Schema.(
+      obj("project", ~doc="Project of an organization", ~fields=_ =>
+        [
+          field("id", ~typ=non_null(int), ~args=Arg.[], ~resolve=(info, p: t) =>
+            p.id
+          ),
+          field(
+            "name",
+            ~typ=non_null(string),
+            ~args=Arg.[],
+            ~resolve=(info, p: t) =>
+            p.name
+          ),
+          field(
+            "webhook",
+            ~typ=non_null(string),
+            ~args=Arg.[],
+            ~resolve=(info, p: t) =>
+            p.webhook
+          ),
+        ]
+      )
+    );
+
+  module Config = {
+    type nodeType = t;
+    type context = Context.t;
+    let nodeResolver = resolver;
+    let nodeName = "project";
+  };
+
+  module Connection = Connection.Create(Config);
+};
+
+module Member = {
+  type t = {
+    id: int,
+    email: string,
+  };
+
+  let resolver =
+    Schema.(
+      obj("member", ~doc="Member of an organization", ~fields=_ =>
+        [
+          field("id", ~typ=non_null(int), ~args=Arg.[], ~resolve=(info, p) =>
+            p.id
+          ),
+          field(
+            "email", ~typ=non_null(string), ~args=Arg.[], ~resolve=(info, p) =>
+            p.email
+          ),
+        ]
+      )
+    );
+
+  module Config = {
+    type nodeType = t;
+    type context = Context.t;
+    let nodeResolver = resolver;
+    let nodeName = "member";
+  };
+
+  module Connection = Connection.Create(Config);
+};
+
+type t = {
+  id: int,
+  name: string,
+  createdAt: string,
+};
+
+let resolver =
+  Schema.(
+    obj("organization", ~doc="User organization", ~fields=_ =>
+      [
+        field(
+          "id",
+          ~typ=non_null(int),
+          ~args=Arg.[],
+          ~resolve=(info: Context.t, p: t) =>
+          p.id
+        ),
+        field(
+          "name",
+          ~typ=non_null(string),
+          ~args=Arg.[],
+          ~resolve=(info: Context.t, p: t) =>
+          p.name
+        ),
+        field(
+          "createdAt",
+          ~typ=non_null(string),
+          ~args=Arg.[],
+          ~resolve=(info: Context.t, p: t) =>
+          p.createdAt
+        ),
+        field(
+          "members",
+          ~typ=Member.Connection.connectionResolver,
```

Comment:
```suggestion
          Connection.connectionResolver((_info, p, first, after, last, before) =>
          None))
```
can we go even further?

Reply:
pass a callback on this connectionResolver, not getting, why?

Reply:
if not better to change connectionResolver to connectionType

Reply:
it's the resolver right so the user doesn't need to type the arguments again

Reply:
```
        field(
          "createdAt",
          ~typ=non_null(string),
          ~args=Arg.[],
          ~resolve=(info: Context.t, p: t) =>
          p.createdAt
        ),
        Connection.connectionResolver("member", (_info, p, first, after, last, before) =>
          None)),
```

Reply:
it's the type resolver

Reply:
the user can pass additional args if wanted to
```Connection.connectionResolver(~args=[], (_info, p, first, after, last, before) =>
          None))```

Reply:
mapping the type return and etc

Reply:
yea the type would be Connection.connectionResolverType actually to keep the naming more accurate

Reply:
yeah that's why I wrote typeResolver before hahah

Reply:
https://github.com/Astrocoders/summhub-server/pull/7/files#r260462546

Reply:
d